src/opera_tosca_parser/template/node.py

```diff
@@ -61,3 +63,16 @@ def resolve_requirements(self, topology: Topology):
             if requirement_occurrences[r.name] > max_occurrences:
                 raise DataError(f"Too many occurrences found for requirement '{r.name}'. Maximum is: {max_occurrences}")
             r.resolve(topology)
+
+    def is_a(self, typ):
+        return typ in self.types
+
+    @property
+    def instance(self):
```

Comment:
Please insert documentation string stub with the similar style as other functions.